### PR TITLE
Styles update

### PR DESCRIPTION
**Pull request by beccarothwell**

### CODE REVIEW

**Review by jennysivapalan — COMMENTED** [review]
This one loads for me yey! But I can't play the game, as no stat to click.

File: client/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "ES2020",
     "useDefineForClassFields": true,
-    "lib": ["ES2020", "DOM", "DOM.Iterable"],
+    "lib": ["ES2020", "DOM", "DOM.Iterable", "ES2021.String"],
```
Comment (jennysivapalan) [inline client/tsconfig.json:5]: What does this do?

Comment (beccarothwell) [reply client/tsconfig.json:5]: This allows for the use of string.replaceAll() with typescript see error mentioned here https://bobbyhadz.com/blog/typescript-string-replace-all-occurrences

Comment (jennysivapalan) [reply client/tsconfig.json:5]: ooh thanks will take a read

**Comment by jennysivapalan** [comment]
Okay to merge it if it's fixed in a later PR :)

**Review by jennysivapalan — APPROVED** [review]

**Pull request merged**